modules/tensor_mechanics/tests/mohr_coulomb/tests

```diff
@@ -126,6 +133,13 @@
     rel_err = 1.0E-5
     abs_zero = 1.0E-3
   [../]
+  [./uni_axial1_small_strain]
+    type = 'CSVDiff'
+    input = 'uni_axial1_small_strain.i'
+    csvdiff = 'uni_axial1.csv'
+    rel_err = 1.0E-5
+    abs_zero = 1.0E-3
```

Comment:
i'm rushing to get other stuff done, but am a bit confused about this nl_rel_tol thing.  I tried:

  #nl_abs_tol = 1E-4
  nl_rel_tol = 1E-6

(commented-out abs_tol).   Then i get:

Time Step  1, time = 0.05
                dt = 0.05
 0 Nonlinear |R| = 2.040003e+06
      0 Linear |R| = 2.040003e+06
      1 Linear |R| = 4.538429e-07
 1 Nonlinear |R| = 7.093712e-09
      0 Linear |R| = 7.093712e-09
      1 Linear |R| = 1.186610e-21
 2 Nonlinear |R| = 1.042989e-10
      0 Linear |R| = 1.042989e-10
      1 Linear |R| = 2.718840e-23
 3 Nonlinear |R| = 9.719489e-12
 Solve Converged!

OK, good convergence, but shouldn't the "rel_tol" thing have stopped moose at the end of the 1 Nonlinear?   Time Step 3 looks like:

Time Step  3, time = 0.15
                dt = 0.05
 0 Nonlinear |R| = 2.040003e+06
      0 Linear |R| = 2.040003e+06
      1 Linear |R| = 7.317844e-07
 1 Nonlinear |R| = 1.329029e-07
      0 Linear |R| = 1.329029e-07
      1 Linear |R| = 5.152391e-20
 2 Nonlinear |R| = 6.067291e-10
      0 Linear |R| = 6.067291e-10
      1 Linear |R| = 1.300472e-22
 3 Nonlinear |R| = 4.243664e-10
      0 Linear |R| = 4.243664e-10
      1 Linear |R| = 4.437868e-22
 4 Nonlinear |R| = 3.703757e-10
      0 Linear |R| = 3.703757e-10
      1 Linear |R| = 3.689410e-22
 5 Nonlinear |R| = 2.504892e-10
      0 Linear |R| = 2.504892e-10
      1 Linear |R| = 2.441251e-22
 6 Nonlinear |R| = 2.244213e-10
      0 Linear |R| = 2.244213e-10
      1 Linear |R| = 2.003709e-22
 7 Nonlinear |R| = 2.078283e-10
      0 Linear |R| = 2.078283e-10
      1 Linear |R| = 2.010800e-22

Probably getting precision-loss, but why didn't we stop at "1 Nonlinear" ?

Sorry that this is a stupid question - too many other things to do.